cloud_images/centos7/dcos_vol_setup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o nounset
```

Comment:
As per your comment this is unnecessary, what I intended here was a `trap '' PIPE` instead.